src/systems/systems/src/overhead/mod.rs

```diff
@@ -448,6 +448,62 @@ impl SimulationElement for MomentaryPushButton {
     }
 }
 
+pub struct MomentaryOnPushButton {
+    is_pressed_id: String,
+    is_on_id: String,
+    is_pressed: bool,
+    last_pressed_state: bool,
+    is_on: bool,
+}
+impl MomentaryOnPushButton {
+    pub fn new(name: &str) -> Self {
+        Self {
+            is_pressed_id: format!("OVHD_{}_IS_PRESSED", name),
+            is_on_id: format!("OVHD_{}_IS_ON", name),
+            is_pressed: false,
+            last_pressed_state: false,
+            is_on: false,
+        }
+    }
+
+    pub fn is_pressed(&self) -> bool {
+        self.is_pressed
+    }
```

Comment:
Can remain private until we need it elsewhere.
```suggestion
    fn is_pressed(&self) -> bool {
        self.is_pressed
    }
```